libvast/vast/detail/recursive_size.hpp

```diff
@@ -36,7 +36,12 @@ recursive_size(const std::filesystem::path& root_dir) {
     return caf::make_error(ec::filesystem_error, err.message());
   for (const auto& f : dir) {
     if (f.is_regular_file()) {
-      const auto size = f.file_size();
+      const auto size = f.file_size(err);
```

Comment:
Sorry about this bug. My fault 😞

Reply:
No worries, we missed it during review. Your PRs are super appreciated!